docs/00-engineering/engineering-roles.md

```diff
@@ -247,50 +249,154 @@ Responsible for theme code and site configuration, ensuring that:
+  * **Drupal Behaviors**. Understands how to attach JavaScript as a scoped behavior.
 
-* **Backend Technologies** You are experienced in programing or scripting with server-side language, server architecture, database administration, security, and system design.
+  * **Drupal Breakpoints**. Understands how breakpoints and image styles are used to promote consistency through an application.
 
-* **Front End Technologies** You are experienced in web standards, markup, web languages, and stay up to date on new Front End technologies. You understand the challenges to over-engineering client-side solutions and how to avoid common pitfalls. You value performance, accessibility, and reusability.
+  * **Drupal CKEditor**. Understands how to configure CKEditor within the drupal framework.
 
-* **Web Design Systems** You know how to quickly understand Design Systems and how to integrate them within projects. You participate in customizing designed components and adding
+  * **Drupal CLI**. Understands how Drush and Drupal Console can be use to deploy changes, assist in migrations, and debugging applications.
 
-* **Drupal Application Design** You lead the effort in the applications quality and development workflow. You support or implement processes and continuous integration efforts. You architect and maintain the core code and varying environments for content production and iterative development. You often focus on stability and security, guiding the overall improvement of each implementation.
+  * **Drupal Custom Entities**. Understanding how and why to create a custom entity in drupal.
 
-* **JavaScript Application Design** You lead the effort in the applications quality and development workflow. You support or implement processes and continuous integration efforts. You architect and maintain the modular component compositions that create performant and stable user interfaces with accessibility and scalability in mind. You are sometimes called to guide conversations around progressive decoupling and how/or when to integrate them in with monolithic content systems.
+  * **Drupal Custom Fields**. Understanding how and why to create a custom field type in drupal.
 
-* **Data flow and APIs** You take charge of how data is stored, maintained, and delivered to the user interface. You understand how to securely consume, and change data received from user interaction. Often times you are called upon to develop custom presentations and workflows from multiple data sources.
+  * **Drupal Form API**. Understands how to utilize Form API.
 
-* **Site Building and Module Configuration** You are comfortable developing low-code page content structures and providing value with implementing and configuring community provided modules. It is your responsibility to maintain these configurations and the quality of any change request.
+  * **Drupal jQuery**. Understands Drupal's implementation of jQuery and how to utilize behaviors.
```

Comment:
Behaviors already has its own entry on 293. Maybe these can be combined or 307 is sufficient without needing 293?